src/components/Thought.tsx

```diff
@@ -179,7 +179,16 @@ const ThoughtContainer = ({
     attributeEquals(state, head(rootedParentOf(state, parentOf(simplePath))), '=view', 'Table'),
   )
 
-  const hideBullet = useHideBullet({ children, env, hideBulletProp, isEditing, simplePath, thoughtId })
+  const isInContextView = useSelector(state => {
+    /** Check if the thought is in the context view. */
+    const checkContextView = (currentPath: Path): boolean => {
+      if (currentPath.length <= 1) return false
+      return isContextViewActive(state, currentPath) || checkContextView(parentOf(currentPath))
```

Comment:
I don't believe this should be recursive. Only the parent thought needs to be checked to determine if a thought is in the context view.